README.md

```
@@ -58,6 +96,31 @@
+
+**Полезные команды:**
+```bash
+pnpm lint              # Проверка кода (ESLint)
+pnpm lint:fix          # Автоисправление стиля
+pnpm test              # Запуск unit-тестов
+pnpm test:watch        # Интерактивный режим тестов
+pnpm test:coverage     # Отчет о покрытии кода
+```
+
+**Структура проекта:**
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)